packages/stacked_crashlytics/README.md

```diff
@@ -0,0 +1,14 @@
+# stacked_crashlytics
+
+A new Flutter package project.
```

Comment:
Remove the basic flutter readme. Instead add some basic details on how to use. 

Make the title 

#Stacked Crashlytics Service

A service accompanied with a LogOutput that tracks all warnings as non-fatal crash reports and all errors as fatal crash reports. This makes use of the Firebase Crashlytics service.

## How To use

- Go over adding the package
- Go over adding the service to the locator. 
- Show where to add the log output

And that should be good enough.